File: lib_ebook_db/src/main/java/com/ebook/db/entity/LibraryKindBookList.java
package com.ebook.db.entity;

import java.util.List;

/**
 * 书城 书籍分类推荐列表
 */
public class LibraryKindBookList {
    private String kindName;
    private String kindUrl;
    private List<SearchBook> books;

    public String getKindName() {
        return kindName;
    }

    public void setKindName(String kindName) {
        this.kindName = kindName;
    }

    public List<SearchBook> getBooks() {
        return books;
    }

    public void setBooks(List<SearchBook> books) {
        this.books = books;
    }

    public String getKindUrl() {
        return kindUrl;
    }

    public void setKindUrl(String kindUrl) {
        this.kindUrl = kindUrl;
    }
}

File: lib_ebook_db/src/main/java/com/ebook/db/entity/LibraryKindBookList.kt
package com.ebook.db.entity

/**
 * 书城 书籍分类推荐列表
 */
data class LibraryKindBookList(
    @JvmField
    var kindName: String = String(),

    @JvmField
    var kindUrl: String = String(),

    @JvmField
    var books: List<SearchBook> = mutableListOf()
)
